recipes/server.rb

```diff
@@ -29,6 +30,20 @@
   notifies :restart, "service[#{node['nfs']['service']['server']}]"
 end
 
+# RHEL7 has some extra requriements per
+# https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/nfs-serverconfig.html#s2-nfs-nfs-firewall-config
+if rhel? && node['platform_version'].to_f >= 7.0
+  include_recipe 'sysctl::default'
```

Comment:
@joerocklin this does not seem to work, unless sysctl::default is actually in the run_list, due to svanzoest-cookbooks/sysctl#19.  I think we should set necessary sysctl parameters, but am wondering if there is a way to do this without putting responsibility on the end user to change their run_list?